### PR TITLE
Started WebStorm Plugin

### PR DESCRIPTION
**Pull request by TomPed**
Started the plugin to jump from source to spec file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# cesium-webstorm-plugin
\ No newline at end of file
+# Cesium WebStorm Plugin
+
+[Download the plugin](https://plugins.jetbrains.com/plugin/8391?pr=).
```
Comment (pjcozzi) [inline README.md]: Include more instructions.  Assuming I don't know how to use Webstorm plugins, what exactly do I do?  Be concise.

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
-# cesium-webstorm-plugin
\ No newline at end of file
+# Cesium WebStorm Plugin
```
Comment (pjcozzi) [inline README.md:1]: If you want to really kill it, include an animated gif that shows swapping between source and spec files.

Comment (pjcozzi) [inline README.md:1]: Also say that this is developed by the Cesium team and include a logo, for example, https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/README.md

**Comment by pjcozzi** [comment]
For the LICENSE.md file, just link to the Apache license using a format like Cesium: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md

Copyright "Analytical Graphics, Inc. and Contributors"

File: README.md (hunk 2 above)
Comment (pjcozzi) [inline README.md:1]: Also provide concise instructions for contributing (really just the CLA) like this: https://github.com/AnalyticalGraphicsInc/gltf-pipeline#contributions

Most likely I don't expect contributions here, but we should welcome them.

**Comment by TomPed** [comment]
Updated. Would it be a good idea to write a build guide for running and testing the plugin?

**Comment by pjcozzi** [comment]
> Would it be a good idea to write a build guide for running and testing the plugin?

Since this is a small project, just add a short section to README.md

**Comment by pjcozzi** [comment]
Can one more person test installing this?  @JoshuaStorm?

**Comment by TomPed** [comment]
Added the guide.

**Comment by TomPed** [comment]
@JoshuaStorm try [this](https://drive.google.com/open?id=0ByHqaF0tPigoZUQzXy1OSmtVVU0) out. [New](https://drive.google.com/open?id=0ByHqaF0tPigoaFFTMVAzOHc2MlE).

**Comment by JoshuaStorm** [comment]
Looks good. Working on both Cesium and gltf-pipeline projects.

**Comment by TomPed** [comment]
I'll push it now.

**Comment by TomPed** [comment]
@lasalvavida, this might be of use to you.

**Comment by TomPed** [comment]
Making some more tweaks.

**Comment by TomPed** [comment]
@pjcozzi good for a look.

**Comment by pjcozzi** [comment]
Go ahead and merge; I don't have commit access.

**Comment by TomPed** [comment]
@pjcozzi @JoshuaStorm thanks!

**Comment by pjcozzi** [comment]
Also, can you please post this to the Cesium forum?

**Pull request merged**